instrumentation/ktor/ktor-1.0/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v1_0/KtorServerTracing.kt

```diff
@@ -18,12 +18,7 @@ import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor
 import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter
 import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor
 import io.opentelemetry.instrumentation.api.instrumenter.SpanStatusExtractor
-import io.opentelemetry.instrumentation.api.instrumenter.http.HttpRouteHolder
-import io.opentelemetry.instrumentation.api.instrumenter.http.HttpRouteSource
-import io.opentelemetry.instrumentation.api.instrumenter.http.HttpServerAttributesExtractor
-import io.opentelemetry.instrumentation.api.instrumenter.http.HttpServerMetrics
-import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanNameExtractor
-import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanStatusExtractor
+import io.opentelemetry.instrumentation.api.instrumenter.http.*
```

Comment:
is using wildcard import here intentional?

Reply:
Huh, nope, I assume that IntelliJ did this on its own. Will fix that, thanks!